changelogs/unreleased/gh-9643-credentials-from-app-roles.md

```diff
@@ -0,0 +1,3 @@
+## feature/config
+
+* A non-existent role can now be assigned via config.
```

Comment:
```suggestion
* A non-existent role can now be assigned via the configuration.
```

Reply:
Thank you! I applied, except that I removed the last word - `file`, since the configuration can be provided in other ways.